lib/src/layout/layout_modern.dart

```diff
@@ -338,116 +338,69 @@ class _LayoutModernState extends State<LayoutModern> {
                                       ),
                                     ),
                                     Config.spaceBox(Config.smallSpacer),
-                                    AnimatedShakingBuilder(
-                                      autoPlay: isDragged,
-                                      child: SizedBox(
-                                        width: widget.w,
-                                        child: Row(
-                                          mainAxisAlignment:
+                                    for(int i=0;i<widget.data.experience.length;i++)
```

Comment:
can you please explain me that approach ?

Reply:
Here we can use a list view as well but this is very easy and efficient and clean